.builds/ubuntu.yml

```diff
@@ -1,4 +1,4 @@
-image: ubuntu/latest
+image: ubuntu/next
```

Comment:
`ubuntu/lts` is probably better than `ubuntu/next` for a CI. I guess that they will switch over to 21.04 devel soon.
See https://man.sr.ht/builds.sr.ht/compatibility.md#ubuntu

Reply:
I like living on the edge, but it is probably good to test with a distro that isn't on the latest version of everything.

I will switch to lts, then.